baselines/teams.md

```diff
@@ -339,34 +388,27 @@ Blog](https://techcommunity.microsoft.com/t5/microsoft-teams-blog/skype-for-busi
 
 ### Implementation
 
-Instructions for *enabling* communications with Skype users are outlined
-in [Communicate with Skype
-users](https://docs.microsoft.com/en-us/microsoftteams/manage-external-access#communicate-with-skype-users).
+All the settings in this section are configured in the **Microsoft Teams admin center**.
```

Comment:
For similar statements made for MS.TEAMS.1 through MS.TEAMS.5, "Microsoft Teams admin center" was a link. Recommend changing this to a link for consistency.

Reply:
Updated to add a link.